### ISSUE TITLE
Consume NPM updates from replicate.npmjs.com instead of RSS

### ISSUE THREAD

**Issue by calebbrown** [ossf/package-feeds#171]
https://replicate.npmjs.com/_changes is the CouchDB endpoint for consuming a feed of database changes. (see https://github.com/npm/registry/blob/master/docs/follower.md and https://docs.couchdb.org/en/3.2.0/api/database/changes.html)

The NPM RSS XML feed regularly has encoding issues:
```
time="2022-03-19T09:15:02Z" level=error msg="Error fetching packages" error="XML syntax error on line 69: illegal character code U+0000" feed=npm
```

Consuming the updates directly from the registry change API will:

1. ensure that encoding issues aren't encountered
2. ensure that *all* changes can be parsed

It isn't clear when new records are added, so there may be more updates in this feed than simply new packages. A filter may be required to limit results to only those we are interested in.

**Comment by calebbrown** [comment]
Example feed URL: https://replicate.npmjs.com/_changes?since=now&feed=continuous&heartbeat=true&include_docs=true

**Comment by maxfisher-g** [comment]
It seems like replicate feed is not suitable due to a time delay, so we will just have to improve the RSS feed as best as possible.

There are three ways to improve the reliability of the RSS feed

1. Increase the number of RSS feed results returned, to avoid missing as many packages
2. Reducing time between polling
3. Dealing with unicode errors - character encoding errors probably occur in the description field. Can look into iconv  library or similar to do this.

**Comment by maxfisher-g** [comment]
This is addressed in #312 with the exception of reducing the polling time / increasing polling rate, which can be done via config change.
